app/furniture/marketplace/products/_product.html.erb

```diff
@@ -13,6 +13,7 @@
     <%= humanized_money_with_symbol(product.price) %>
   </td>
   <td class="py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-6">
+    <%= render "buttons/edit_pencil", title: t('.edit'), href: [:edit]+product.location, method: :get %>
```

Comment:
Can we name this just `buttons/edit` since the actual icon may vary depending on the Space's theme or passed in parameter? Then we can pass `label: t('icons.pencil')` if we want to use something other than the `⚙️`?

Reply:
I possibly fixed it via https://github.com/zinc-collective/convene/pull/942/commits/0a99dceb62945a62b110a009e090ae06e46c9675

I can't test on local if the pencil icon still displays because merging latest master changes to `products_controller.rb` throws a lovely `Pundit::AuthorizationNotPerformedError` when clicking `Manage Products` link on for a Marketplace furniture.

<img width="1680" alt="Screen Shot 2022-11-20 at 12 48 23 PM" src="https://user-images.githubusercontent.com/16140873/202925702-247b4411-2fe6-4d5b-83b9-07dbeec4d948.png">

Let's fix that in Wednesday 11/23 session.

Reply:
Yep, looks like we didn't include Policy objects in our earlier work; and since we didn't have request specs on the Products Controller, so my PR to [require authorize calls on all requests](https://github.com/zinc-collective/convene/pull/929) didn't trip.

When we add feature or request specs to the Marketplace that should prevent that kind of thing going forward.